examples/job/create_and_submit_job.py

```diff
@@ -30,7 +34,9 @@
 import os, glob, sys, importlib, urllib.request
 
 # The below execution sets up runtime using code stored remotely in a url
-exec(urllib.request.urlopen('https://raw.githubusercontent.com/Exabyte-io/exabyte-api-examples/dev/examples/utils/initialize_settings.py').read())
+if 'isExecuted' not in globals():
```

Comment:
This doesn't seem like a great solution - having to add if to every notebook. I'd just modify https://github.com/Exabyte-io/exabyte-api-examples/blob/dev/examples/utils/initialize_settings.py#L56 instead and change the environment variable `is_setup_executed` therein or something like that.